patches/net/minecraft/world/level/storage/LevelSummary.java.patch

```diff
@@ -5,7 +5,7 @@
      }
  
 +    public boolean isLifecycleExperimental() {
-+        return this.settings.getLifecycle().equals(com.mojang.serialization.Lifecycle.experimental());
++        return this.settings != null && this.settings.getLifecycle().equals(com.mojang.serialization.Lifecycle.experimental());
 +    }
```

Comment:
Since the `LevelSettings` is now accessible through `LevelSummary#getSettings()`, should we remove this patched-in method and inline that in the one place it's used (in `WorldSelectionList.WorldListEntry`).